src/main/java/by/onliner/page/ShoppingCartPage.java

```diff
@@ -54,11 +75,71 @@ public String getRemovedItemInformation() {
     /***
      * Get a message from empty cart.
      */
-    @Step("Read the empty cart message")
+    @Step("Get the empty cart message")
     public String getEmptyCartMassage() {
         DriverManager.getDriver().navigate().refresh();
         Waiter.waitForVisibility(emptyCartMessage);
         logger.info(String.format("Displayed [%s] message", emptyCartMessage.getText()));
         return emptyCartMessage.getText();
     }
+
+    /***
+     * Get the item name from shopping cart list.
+     * @return name of the item
+     */
+    @Step("Get the item name")
+    public String getItemNameText() {
+        Waiter.waitForVisibility(itemNameText);
+        logger.info(String.format("Read the item name [%s] from shopping cart", itemNameText.getText()));
+        return itemNameText.getText().trim();
+    }
+
+    /***
+     * Check the complete order button fo visibility.
+     * @return visibility of button
+     */
+    @Step("Chek [Перейти к оформлению] button for visibility")
+    public boolean isCompleteOrderButtonVisible() {
+        boolean visibility = completeOrderButton.isDisplayed();
+        logger.info(String.format("'Перейти к оформлению' button visibility is {%s}", visibility));
+        return visibility;
+    }
+
+    /***
+     * Add one more same item in the shopping cart by [+] button
+     */
+    @Step("Add one more same item in the shopping cart by [+] button")
+    public void clickQuantityInputPlusButton() {
+        Waiter.elementToBeClickable(quantityInputPlusButton);
+        quantityInputPlusButton.click();
+        logger.info("Click '+' button");
+        Waiter.sleep(2);
```

Comment:
I thought we agreed to implement method using awaitility to wait until price is changed, didn't we?

Reply:
Implement Awaitility